client/xamarin.forms/iOS/packages.tt

```diff
@@ -4,10 +4,12 @@
 <#@parameter type="System.String" name="StoreSDKVersion"#>
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="xamarinios10" />
-  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="xamarinios10" />
-  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="xamarinios10" />
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="xamarinios10" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="xamarinios10" />
+  <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="xamarinios10" />
+  <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="xamarinios10" />
+  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="xamarinios10" />
+  <package id="SQLitePCL" version="3.8.7.2" targetFramework="xamarinios10" />
   <package id="WindowsAzure.MobileServices" version="<#= ManagedSDKVersion #>" targetFramework="xamarinios10" />
+  <package id="WindowsAzure.MobileServices.SQLiteStore" version="<#= ManagedSDKVersion #>" targetFramework="xamarinios10" />
```

Comment:
Updated all SQLiteStore references with the correct property name (25492a8e411f895a6291f75b1dbc0645678f43ad).